pastas/stats/core.py

```diff
@@ -118,12 +127,13 @@ def ccf(
     x: Series,
     y: Series,
     lags: ArrayLike = 365,
-    bin_method: str = "rectangle",
+    bin_method: str = "gaussian",
```

Comment:
How come we use "gaussian" as default here rather than "regular" like in the `acf()` function?

Reply:
That should be regular here indeed!